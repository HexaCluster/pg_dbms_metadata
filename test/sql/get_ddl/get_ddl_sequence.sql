SELECT dbms_metadata.get_ddl('SEQUENCE','attach_line','gdmmm');