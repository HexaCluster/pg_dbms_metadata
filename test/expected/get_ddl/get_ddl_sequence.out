                                                         get_ddl                                                         
-------------------------------------------------------------------------------------------------------------------------
 CREATE SEQUENCE gdmmm.attach_line START WITH 1 INCREMENT BY 1 MINVALUE 1 MAXVALUE 9223372036854775807 CACHE 1 NO CYCLE;
(1 row)

